MIGRATING-0.6-0.7.md

```diff
@@ -331,10 +365,30 @@ It now also disallows out of bounds drawing by default. This behaviour can be ch
 
 ## Style module
 
-The `style` module has been removed.
+The `style` module has been removed. The items in it have been moved:
 
-TODO: Improve this section before release.
+- `PrimitiveStyle` and `PrimitiveStyleBuilder` are now available under `embedded_graphics::primitives::{PrimitiveStyle, PrimitiveStyleBuilder}`.
+- `TextStyle` and `TextStyleBuilder` were renamed are now available under `embedded_graphics::mono_font::{MonoTextStyle, MonoTextStyleBuilder}`.
+
+  Note that usage with `Text` has changed. See [the text changes section](#Text-rendering) for more.
+
+## Text and fonts
+
+The collection of builtin fonts are now sourced from public domain BDF fonts in the XOrg project.
+Due to this, they have slightly different dimensions and glyphs and so have changed names. Some
+sizes are not the same in the new set, but a rough mapping is as follows:
+
+| Old font                                                                                                                                                                             | Visually closest new font                                                                                                                                                                                      |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `fonts::Font6x6`<br>![Font6x6 glpyh bitmap](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/embedded-graphics-v0.6.2/embedded-graphics/data/font6x6.png)       | `mono_font::[ascii/latin1]::FONT_4X6`<br>![FONT_4X6 glyph bitmap](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/embedded-graphics-v0.7.0-beta.1/fonts/ascii/png/4x6.png)               |
```

Comment:
We use `[ascii|latin1]` above and here we use `[ascii/latin1]`. I'm not too concerned which one we use, but I think we should use the same throughout the document. Or we could switch to a syntax similar to a use statement: `mono_font::{ascii, latin1}::FONT_4X6` or `mono_font::{ascii | latin1}::FONT_4X6`.

Reply:
I changed this to work with Markdown tables - my autoformatter gets confused if I use a pipe. I've changed it to `{ascii, latin1}` which looks a lot better than either previous solution - thanks.